### DISCUSSION TITLE
Fork from repository using main to a organnistion configure for master issue

### DISCUSSION THREAD

**Discussion by darkedges** [community/community#22422] [New to GitHub]
<p><span class="mention">@github</span> any idea why when I forked this repo <a href="https://t.co/gihtgXpghH" class="inline-onebox" rel="noopener nofollow ugc">https://github.com/hashicorp/vault/</a> to <a href="https://t.co/MqE5yXOGYx" class="inline-onebox" rel="noopener nofollow ugc">https://github.com/darkedges/vault/</a> they are not in sync.</p>
<p>Is this a main vs master thing?<br>
Regards<br>
DarkEdges</p>

**Comment by kingthorin** [comment]
<p>It seems to be from my perspective except for 2 commits that are more recent.</p>

**Comment by darkedges** [comment, marked as answer]
<p>After updating to using main instead of master in my settings I was able to fork properly.</p>
<p>Going to close.</p>
